Cannot build on Arch Linux

[ 72%] Building CXX object src/CoreFoundation/CMakeFiles/CoreFoundation.dir/AppleCFString.o
/home/ling/PKGS/darling/13_darling-git/src/darling/src/CoreFoundation/AppleCFString.mm:8:64: error: definition
      '__CFConstantStringClassReference' cannot also be an alias
Class* __CFConstantStringClassReference  __attribute__ ((weak, alias ("_OBJC_CLASS_AppleCFString")));
                                                               ^
1 error generated.
src/CoreFoundation/CMakeFiles/CoreFoundation.dir/build.make:86: recipe for target 'src/CoreFoundation/CMakeFiles/CoreFoundation.dir/AppleCFString.o' failed

--- a/src/CoreFoundation/AppleCFString.mm
+++ b/src/CoreFoundation/AppleCFString.mm
@@ -5,7 +5,7 @@
 extern "C" Class* _OBJC_CLASS_AppleCFString;
 extern "C" Class* __CFConstantStringClassReference;
 
-Class* __CFConstantStringClassReference  __attribute__ ((weak, alias ("_OBJC_CLASS_AppleCFString")));
+extern Class* __CFConstantStringClassReference  __attribute__ ((weak, alias ("_OBJC_CLASS_AppleCFString")));
 
 __attribute__((constructor)) static void forceAppleCFStringLoad()
 {
@@ -36,7 +36,7 @@ __attribute__((constructor)) static void forceAppleCFStringLoad()
 {
 	if (aRange.location+aRange.length > _length)
 		[NSException raise: NSRangeException format: @"Invalid range"];
-	
+
 	if (_flags & apple_unicode)
 	{
 		memcpy(buffer, _data._unicodeData + aRange.location, aRange.length);

